fast64_internal/oot/oot_utility.py

```diff
@@ -874,6 +874,27 @@ def _eval(node):
     return f"0x{_eval(node.body):X}"
 
 
+def getShiftFromMask(mask: int):
+    """Returns the shift value from the mask"""
+
+    # get the shift by subtracting the length of the mask
+    # converted in binary on 16 bits (since the mask can be on 16 bits) with
+    # that length but with the rightmost zeros stripped
+    return int(f"{len(f'{mask:016b}') - len(f'{mask:016b}'.rstrip('0'))}", base=10)
+
+
+def getFormattedParams(mask: int, value: str, isBool: bool):
```

Comment:
can mask be None ? the function partly does and doesn't handle `mask is None`

Reply:
mask is never ``None``, this is how mask is parsed from the xml: ``elem.get("Mask", "0xFFFF")``, if there's no mask defined it defaults to 0xFFFF ([line's here](https://github.com/Yanis42/fast64/blob/acfe3619860ae0f34f62ee5b01b7d85f9c8cda33/fast64_internal/oot/data/oot_actor_data.py#L97))

I'm not sure why there's ``if mask is not None else 0`` in that function :thonk: